website/community/product-steering/_index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Product Steering
+weight: 10
+---
+
+## Product Steering
+
+### What is The Product Steering?
+Product Steering provides strategic direction for major product increments. Product ideas can be submitted to the Product Steering Committee as proposals. The proposer must provide a brief document outlining the idea. Ideally, if applicable, a prototype should be included to demonstrate feasibility, but no full implementation or design document is expected (or desired) at this stage.
+
+The committee will schedule a Product Steering meeting, where the proposer presents the proposal. Committee members, interested contributors, and affected sponsors will challenge and discuss the idea. This meeting will be recorded.
+
+Following this, a private go/no-go decision meeting with the affected sponsors will take place. This second meeting will not be recorded, but the outcome will be documented and shared.
+
+Only proposals requiring significant development effort or broad cross-team impact will be considered.
+
+### Why Do We Need It?
+We aim to align sponsored development capacity with Gardener's strategic vision and priorities. Investments should deliver value either to Gardener's users and operators or to the project and its overall reach.
+
+### Who is on The Product Steering Committee?
+The committee is elected by the project's active contributing sponsors and consists of a small, odd number of members (at least three) to ensure majority decision-making. Currently, the committee members are:
+- [Johannes Scheerer](https://github.com/scheererj)
+- [Rafael Franzke](https://github.com/rfranzke)
+- [Vedran Lerenc](https://github.com/vlerenc)
+
+### How Does It Work?
+#### 1. Proposal Submission
+- A proposer drafts a **2-5 page product proposal** detailing the idea and shares it with any member of the Product Steering Committee.
```

Comment:
How is this proposal shared with the committee?

Reply:
I will mention both.